bet-predictor/cmd/update_teams.go

```diff
@@ -0,0 +1,144 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/serhiihuberniuk/bet-predictor/models"
+	"github.com/serhiihuberniuk/bet-predictor/scanner"
+	"github.com/spf13/cobra"
+)
+
+var leagueFlag string
+var countryFlag string
+
+var updateTeamsCmd = &cobra.Command{
+	Use:   "update:teams",
+	Short: "Sync teams in your DB",
+	Long:  "Long description of update:teams",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		commandContext, err := commandInit()
+		defer commandContext.DisconnectFn()
+		defer commandContext.CancelFn()
```

Comment:
thy this is not one function?

Reply:
it's not correct to create the close functions in a context in this way: it's not clear what should be called after initialization

Reply:
you do defer even when error happend